examples/angular/src/pages/ui/components/authenticator/sign-up-with-email-lambda/sign-up-with-email-lambda.component.html

```diff
@@ -4,7 +4,7 @@
     let-user="user"
     let-signOut="signOut"
   >
-    <h2>Welcome, {{ user.username }}!</h2>
```

Comment:
The test for this example was relying on username, even though this is testing email. I changed the test to better reflect what the user should be seeing.

Reply:
IMO it's okay to use `user.username`, because username is email when your cognito backend has email as login mechanism.

Let's keep `user.username`, since that's what we prefer in docs as well.